src/lumen/web/server.cpp

```diff
@@ -40,23 +37,134 @@ Server::~Server()
 
 namespace {
 
+/*
+ *
+ */
+bool check_file_extension(char const* filepath, char const* ext)
```

Comment:
I think we should update this to use C++ before we merge it into `dev`.

Reply:
Since it is functional, I think updating it to C++ can be handled in its own issue.